src/App.ts

```diff
@@ -89,6 +91,21 @@ async function startServer(servicePort: number) {
   app.listen(servicePort, () =>
     console.log('App listining on port', servicePort)
   )
+
+  app.post(
+    '/api/events/add',
+    authorizeRequest,
+    async (req, res) => {
+      try {
+        const field = await calendarEventService.createCalendarEvent(req.body)
```

Comment:
Käyttäjältä saatu data (`req.body`) pitää kyllä jotenkin validoida, ennen kuin sitä päästää eteenpäin. Suosittelisin ottamaan projektiin mukaan [Zod](https://github.com/colinhacks/zod)-kirjaston ja käyttämään sitä.